### PR TITLE
Update Get-CsPhoneNumberAssignment.md

### PR DESCRIPTION
**Pull request by JensTrier**
Update to add more information and example for Shared Calling.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 1]
```diff
@@ -138,6 +138,12 @@ Get-CsPhoneNumberAssignment -Top ([int]::MaxValue)
 ```
 This example returns all phone numbers.
 
+### Example 11
+```powershell
+Get-CsPhoneNumberAssignment -AssignedPstnTargetId 'TeamsSharedCallingRoutingPolicy|Tag:SC1'
+```
+This example returns all phone numbers assigned as emergency numbers in the Teams Shared Calling Routing Policy instance SC1.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:145]:
```suggestion
This example returns all phone numbers assigned as emergency numbers in the Teams shared calling routing policy instance SC1.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 2]
```diff
@@ -156,7 +162,7 @@ Accept wildcard characters: False
 ```
 
 ### -AssignedPstnTargetId
-Filters the returned results based on the user or resource account ID the phone number is assigned to. Supported values are UserPrincipalName, SIP address, and ObjectId.
+Filters the returned results based on the user or resource account ID the phone number is assigned to. Supported values are UserPrincipalName, SIP address, ObjectId and the Teams Shared Calling Routing Policy instance name.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:165]:
```suggestion
Filters the returned results based on the user or resource account ID the phone number is assigned to. Supported values are UserPrincipalName, SIP address, ObjectId, and the Teams shared calling routing policy instance name.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**